### DISCUSSION TITLE
How can distinguish OG from NFT plot?

### DISCUSSION THREAD

**Discussion by nicezic** [Chia-Network/chia-blockchain#8663] [Q&A]
How can distinguish OG from NFT plot?

**Comment by ByronAP** [comment]
An NFT plot will have a pool_contract_puzzle_hash where as an OG plot will have a pool_public_key

**Reply by DPRC137** [reply]
Video explaining the same here: https://youtu.be/-mwXKrI6IEY?t=684

**Comment by BrandtH22** [comment]
@nicezic , based on the previous messages it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
